dashboard/app/models/user.rb

```diff
@@ -248,6 +248,8 @@ def delete_course_as_facilitator(course)
   end
 
   def log_admin_save
+    return if new_record? && !admin?
```

Comment:
Actually, should this belong in the `log_admin_save` method or in the `before_save` callback logic?